File: cuaderno1/Introduction_v2/PPrincipal.f90

    program Muchas

     use mpi
     implicit none

     !include "mpif.h"

     integer ierr

     call MPI_INIT(ierr)
     call PasoMensajes_2(MPI_COMM_WORLD)

     call MPI_FINALIZE(ierr)
    end program Muchas

File: cuaderno1/Introduction_v2AL/PPrincipal.c
/*************************************************************
	Introduccion_v2
************************************************************/

#include <mpi.h>
#include <stdio.h>

int main (int argc, char *argv[])
{
     MPI_Init(&argc, &argv);

     PasoMensajes_2(MPI_COMM_WORLD);

     MPI_Finalize();
     return (0);
}

